assets/css/admin/downloads/style.scss

```diff
@@ -0,0 +1 @@
+@import "custom-prices";
```

Comment:
The markup references `custom-prices` but I think calling this `variable-prices` or just `prices` would make mores sense. There is an [EDD Custom Prices](https://github.com/easydigitaldownloads/edd-custom-prices) extension this could be confused with.

Reply:
I would be fine with renaming all of the related classes, maybe to something which matches the text, like `.edd-variable-price-option-advanced-settings` and so on. Since none of the extensions have been updated yet, I think we could be pretty flexible here, but if a third party extension is counting on these styles, which were added in 2.8, then that might be problematic.

Reply:
Update: @spencerfinnell it turns out that All Access is also an affected plugin (it's using the "new" markup, but not the correct hook), so it may be safest to keep the classes as they are, but we could certainly rename the partial file.

Reply:
Yes I think renaming the partial will help avoid confusing the contents with CSS specific to the Custom Prices integration.